docs/monitoring.md

```diff
@@ -190,7 +190,10 @@ Security options for the Spark History Server are covered more detail in the
     <td>1d</td>
     <td>
       How often the filesystem job history cleaner checks for files to delete.
-      Files are only deleted if they are older than <code>spark.history.fs.cleaner.maxAge</code>
+      Files are deleted in two cases. First, they are older than <code>spark.history.fs.cleaner.maxAge</code>.
```

Comment:
Maybe more like, "Files are deleted if at least one of two conditions holds. First, they're deleted if they're older than ... They are also deleted if the number of files is more than ..."

Reply:
Thanks!